src/boot.tsx

```diff
@@ -31,7 +47,8 @@ export default (element: HTMLDivElement, hub: Hub) => {
     const exercise = JSON.parse(atob(decodeURIComponent(element.textContent)));
     settings.hint = exercise.hint;
     settings.language = exercise.language;
-    settings.pre_exercise_code = exercise.pre_exercise_code;
+    settings.lang_version = exercise.lang_version;
```

Comment:
Is it possible to rename `lang_version` to `languageVersion`? Another solution could be to have `{ language: { name: 'python', version: '3.5' } }`

Reply:
I'm fine with changing it from `lang_version` to `languageVersion`, although all the other attributes on this object have underscores instead of camel-case. In fact we have `pre_exercise_code` and `sample_code`, rather than `preExerciseCode` and `sampleCode`. Shall I change all of them or leave it?

Reply:
Leave it :)